git-ftp

```diff
@@ -811,6 +816,45 @@ handle_remote_protocol_options() {
 	[ "$CURL_INSECURE" != "0" ] && REMOTE_CMD_OPTIONS="$REMOTE_CMD_OPTIONS -k"
 }
 
+extract_folder_clone() {
+    local fulladdress=$1
+    #check if the last char is /, in this case eliminate it!
+    length=${#filename}
+    ((length--))
+    [ "${fulladdress:$length:1}" == "/" ] && fulladdress="${fulladdress%?}"
+
+    FOLDER_CLONE=${fulladdress##*/}
+}
+
+set_up_clone_folder() {
+    
+    [ -z $URL ] && echo Error: give a url to clone. && exit
+    [ $FOLDER_CLONE == "." ] && extract_folder_clone $URL
+    
+    if [ -d "$FOLDER_CLONE" ]; then
```

Comment:
I don't think more interactive features are a good idea. It would be better to exit here and give a good error message. Leave the deletion to the user. They may want to rename it or put it into a trash folder. "Overwrite" could also mean, that you copy new files on top of the old ones, but that local files that are not on the server actually persist. That would be a harmful misunderstanding. Better to say:
```
The directory $CLONE_DIR already exists. Aborting.
You can delete that directory if you don't need it any more or give a new directory name on the command line.
```